gridfs/__init__.py

```diff
@@ -848,6 +865,8 @@ def open_download_stream_by_name(self, filename, revision=-1, session=None):
         query = {"filename": filename}
         _disallow_transactions(session)
         cursor = self._files.find(query, session=session)
+        if revision is None:
+          revision = -1
```

Comment:
Why this change? Looking at the API we do not support `revision=None`.

Reply:
the `Optional[int]` implies `None` is allowed.  I removed the `Optional`